### ISSUE TITLE
Incompatible definition of methods reported when using multiple inheritance

### ISSUE THREAD

**Issue by frerich** [python/mypy#9319]
Using mypy 0.782, errors about incompatible method redefinitions are issued -- but only if multiple inheritance is in play.

## How to Reproduce

Run mypy (without any arguments) on this script:

```python
import enum

class C:
    pass

class D(C, enum.IntFlag):
    pass
```

## Expected behaviour
mypy should print

```
Success: no issues found in 1 source file
```

## Actual behaviour
mypy prints three error messages:

```
mypy-error.py:6: error: Definition of "__xor__" in base class "int" is incompatible with definition in base class "Flag"
mypy-error.py:6: error: Definition of "__or__" in base class "int" is incompatible with definition in base class "Flag"
mypy-error.py:6: error: Definition of "__and__" in base class "int" is incompatible with definition in base class "Flag"
Found 3 errors in 1 file (checked 1 source file)
```

### Comments

The error messages go away when

* Not using multiple inheritance, i.e. `class D(enum.IntFlag)` instead of `class D(C, enum.IntFlag)`.
* Or: when using `enum.Flag` instead of `enum.IntFlag`

**Comment by JukkaL** [comment]
This looks like a bug in type checking classes with multiple inheritance. You can use `# type: ignore` on line 6 as a workaround for now.

**Comment by adambirds** [comment]
Is this related to this issue [here](https://github.com/typeddjango/django-stubs/issues/471)? Is there any idea when a proper fix will be available?

**Comment by benbovy** [comment]
Is there any update on this?

Alternatively to adding `# type: ignore`, another workaround is to redefine the problematic methods in the subclass with a simple call to `super`.  That's not pretty either but that prevents ignoring the whole inheritance.
